tests/test_us_verification.py

```diff
@@ -17,6 +17,18 @@ def test_us_verification(self):
         self.assertEqual(addr.deliverability, 'deliverable')
         self.assertEqual(addr.primary_line, '1 TELEGRAPH HILL BLVD')
 
+    def test_us_verification_override_lob_version(self):
+        addr = lob.USVerification.create(
+            primary_line='deliverable',
+            city='San Francisco',
+            state='CA',
+            zip_code='94107',
+            lob_version='2018-03-30'
+        )
+
+        self.assertEqual(addr.deliverability, 'deliverable')
```

Comment:
I guess the tests here are actually making the request and verifying it gets something sane back? Do we have a precedent for testing that the header is set ____ is passed in (`lob_version`, in this case)?